### DISCUSSION TITLE
Ground Items quantity filter

### DISCUSSION THREAD

**Discussion by mystyq** [runelite/runelite#17729] [Ideas]
Add a quantity filter to the Ground Items plugin.  For example, I want to hide the text from a singular chaos rune on the ground that spawns at Moss Giants in the wilderness.  I do not want to hide 100 chaos runes that drop from Bryophyta.

It could be defined with `<` `<=` `>` `>=` `=` as an operator after the name.

An example hidden items list would look something like: `*Bone*,mithril sword,chaos rune<=1`.

**Comment by Alexsuperfly** [comment]
this already exists https://github.com/runelite/runelite/wiki/Ground-Items

**Reply by mystyq** [reply]
Hi, thanks for that.  I see that the `<` and `>` operators exist.  Perhaps the `<=`, `>=`, `=` could be added as well.

I feel like those operators would be more explicit.  Having them would only improve filter, imo.

Would you like me to create a regular issue for this?

**Comment by geheur** [comment]
It would be neat if the default value of hidden items showed `coins<10` to let people know this exists. https://github.com/runelite/runelite/pull/17321

**Comment by cdfisher** [comment]
> Hi, thanks for that. I see that the `<` and `>` operators exist. Perhaps the `<=`, `>=`, `=` could be added as well.
> 
> I feel like those operators would be more explicit. Having them would only improve filter, imo.

For `<=` and `>=` , just change the quantity by +/- 1 and the functionality already exists.

What would be the use case for only showing/hiding an exact stack size that would actually justify adding  `=`?

**Reply by mystyq** [reply]
Adding the extra operators would provide a more intuitive experience for the user.

Having the flexibility to provide more in-depth filters justifies adding it.

I had no idea that this was already implemented even after searching for the issue on Github (it wasn't an issue it was a feature already!).  When I went to try out the idea in game, I couldn't quite put my finger on why `<=`, `>=`, or `=` weren't working.  I didn't even think to try `<` or `>`.

I suppose it is the way my brain processes things.
